triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -419,14 +447,23 @@ class Runner(
   )(implicit materializer: Materializer, executionContext: ExecutionContext): (T, Future[SExpr]) = {
     val (source, postFailure) =
       msgSource(client, offset, trigger.heartbeat, party, transactionFilter)
-    def submit(req: SubmitRequest) = {
-      val f = client.commandClient
+    def submit(req: SubmitRequest): Unit = {
+      // Fire off a comand.
+      val f: Future[Empty] = client.commandClient
         .withTimeProvider(Some(Runner.getTimeProvider(timeProviderType)))
         .submitSingleCommand(req)
+
+      // 'f.failed' is a new future that will be successfully completed
```

Comment:
Let’s please not document the standard library in our own code. If I see an inline comment I expect it to be something that I should pay attention to. There is nothing surprising or unusual here.

Reply:
Ok